include/jemalloc/internal/jemalloc_internal_decls.h

```diff
@@ -3,56 +3,56 @@
 
 #include <math.h>
 #ifdef _WIN32
-#  include <windows.h>
-#  include "msvc_compat/windows_extra.h"
-#  ifdef _WIN64
-#    if LG_VADDR <= 32
-#      error Generate the headers using x64 vcargs
-#    endif
-#  else
-#    if LG_VADDR > 32
-#      undef LG_VADDR
-#      define LG_VADDR 32
-#    endif
-#  endif
+#include <windows.h>
+#include "msvc_compat/windows_extra.h"
+#ifdef _WIN64
+#if LG_VADDR <= 32
+#error Generate the headers using x64 vcargs
+#endif
+#else
+#if LG_VADDR > 32
+#undef LG_VADDR
+#define LG_VADDR 32
+#endif
+#endif
 #else
-#  include <sys/param.h>
-#  include <sys/mman.h>
-#  if !defined(__pnacl__) && !defined(__native_client__)
-#    include <sys/syscall.h>
-#    if !defined(SYS_write) && defined(__NR_write)
-#      define SYS_write __NR_write
```

Comment:
Nested cpp logic gets a lot harder to grok without whitespace....

Reply:
I think this is a config setting actually, I'll double-check tomorrow

Reply:
Note to self for tomorrow: `IndentPPDirectives: AfterHash`